example/road-traffic/road-traffic-rest/src/main/resources/ExampleGraphExportOperations.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I don't think we need this file. We should be able to refer to ExportToOtherGraphOperationDeclarations.json instead in these properties: example/road-traffic/road-traffic-demo/src/main/resources/mockaccumulo.properties